src/app/organizations/collection/collection.component.html

```diff
@@ -133,6 +133,7 @@ <h5>
                       </span>
                     </div>
                   </div>
+                  <span>{{ entry.topic }}</span>
```

Comment:
Suggesting that we make the styling similar to the search page (smaller text, truncate to two lines with the full topic displayed in a tooltip).

We should also make sure we display the AI bubble if it's an unapproved AI topic

![image](https://github.com/user-attachments/assets/f71c359c-b1fb-4915-8200-16f36661a741)